Client/Frontend/Browser/Browser.swift

```diff
@@ -11,8 +11,14 @@ protocol BrowserHelper {
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage)
 }
 
+protocol BrowserDelegate {
+    func snackbarAdded(bar: SnackBar)
```

Comment:
These delegate functions need to include the `Browser` I think, since we have more than one `Browser` instance alive. I think we also need to use the more common naming that `UIKit` uses. So something like:

```
func browser(browser: Browser, didAddSnackbar bar: Snackbar)
func browser(browser: Browser, didRemoveSnackbar bar: Snackbar)
```